src/client/components/Radio/ButtonRadio.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const users = ['Student', 'Mentor', 'Admin'];
+function ButtonRadio() {
+  return (
+    <div className="btn-group">
+      {users.map((u) => (
+        <>
+          <button type = "button" name="user" className="button">
```

Comment:
I will suggest you change the class name **button** to **your-component-name-button** so you have something like **button-radio-styling** or **button-radio-button** .  It might work now but If there is global styling for all buttons it'll affect what you have.

Reply:
i changed the name @STEP0HEN